test/unit/double_free.c

```diff
@@ -21,6 +21,15 @@ test_double_free_post() {
 	safety_check_set_abort(NULL);
 }
 
+bool enable_tcache() {
```

Comment:
nit: Yeah I meant to rename this function to `bool tcache_enabled()`, then the name itself shows its a state checking (i.e. `is tcache enabled?` instead of `this enables tcache`).

Reply:
Changed the function name, also change the name of the internal variable to avoid the function and the variable to have the same name for safety